accord-core/src/test/java/accord/impl/basic/DelayedCommandStores.java

```diff
@@ -232,7 +235,7 @@ public <T> AsyncChain<T> submit(PreLoadContext context, Function<? super SafeCom
         public <T> AsyncChain<T> submit(Callable<T> fn)
         {
             Task<T> task = new DelayedTask<>(fn);
-            if (Invariants.isParanoid())
+            if (Invariants.paranoia() >= 3)
```

Comment:
an issue I have with this is there isn't really any documentation on what levels are allowed or what they mean... this might be a better case for an enum then int level?

Reply:
That’s totally fair - just thought this was a flexible and easy thing but fine with defining some descriptive enums like log levels